ctlra/usb.c

```diff
@@ -1,8 +1,10 @@
+#define _BSD_SOURCE
```

Comment:
Out of interest, what's this for?

Reply:
A remained of testing the wait code with a usleep() to delay the polling for completed events... will fix, thanks for reviewing!